### ISSUE TITLE
The "About", "Hide" and "Quit" items do not really need the "(2023)" postfix

### ISSUE THREAD

**Issue by bertfrees** [daisy/pipeline-ui#134]
Maybe another nit-pick, and others may disagree, but I think the "(2023)" could be used more sparingly as it was only added to be able to make a distinction with Pipeline 1 and the old Pipeline 2 GUI.

On the other hand the "- App" postfix, which was added for a very similar reason, could be used more. Currently it is only used for the window title, but it would also make sense in the dock, in the command-tab list, and in the tray tooltip.

Basically I think whenever the "(2023)" is used the "- App" makes sense, and visa versa.

Anyway, as it's meant to be only a temporary solution it might not be worth our time.

**Comment by marisademeglio** [comment]
Oh I thought we added (2023) to indicate that this is the latest Pipeline, not for assistive tech. Hope we can fade out this qualifier though.

And the " - App" is for screenreaders who read out window titles but not the window type. 

For reference, here are all the places we have the title appear (and I will probably edit this list a lot):

* Installer
* Installed shortcut 
* Dock icon
* Application window title
* Settings window title
* About box window title
* Tray menu tooltip text
* Application menu title (mac only)
* Hide and Quit menu items (mac only)
* About menu item

I would prefer in the future to use only "DAISY Pipeline" for everything except the application window title (and maybe the settings & about titles). Those can be " - App".

**Comment by marisademeglio** [comment]
Putting this under the 1.0 milestone because I'd like to use our (2023) workaround as lightly as possible since we want to move past it in the future.

**Comment by marisademeglio** [comment]
how about

  | 2023 | App
-- | -- | --
Installer | yes | **no**
Installed shortcut | yes | **no**
Dock icon | yes | **no**
Application window title | yes | yes
Settings window title | yes | yes
About box window title | yes | yes
Tray menu tooltip text | yes | **no**
Application menu title (mac only) | no | no
Hide and Quit menu items (mac only) | no | no
About menu item | no | no
About box heading | **yes** | no

**Comment by marisademeglio** [comment]
I do not think there is a way to modify the application menu title without the change affecting the installer, shortcut, etc. So it's going to stay as "DAISY Pipeline (2023)"
